src/zac/conf/local_example.py

```diff
@@ -5,8 +5,10 @@
 import os
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-DJANGO_PROJECT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
-BASE_DIR = os.path.abspath(os.path.join(DJANGO_PROJECT_DIR, os.path.pardir, os.path.pardir))
+DJANGO_PROJECT_DIR = os.path.abspath(
```

Comment:
I've just pushed a commit adding the linter configuration (27bf05c6d3e938091ce864e8af2308edda814247) since we stick to line lengths of 120 characters and probably VS auto-formatted this.